parsons/databases/db_sync.py

```diff
@@ -133,35 +133,41 @@ def table_sync_incremental(self, source_table, destination_table, primary_key,
         dest_max_pk = destination_tbl.max_primary_key(primary_key)
 
         # Check for a mismatch in row counts.
-        if dest_max_pk > source_max_pk:
+        if dest_max_pk and dest_max_pk > source_max_pk:
```

Comment:
Looks like you are checking `dest_max_pk` twice.

Reply:
So, I was getting an error if the `dest_max_pk` was `None`: `TypeError: '>' not supported between instances of 'NoneType' and 'int'`

I actually think I need to change this though to `if dest_max_pk is not null and dest_max_pk > source_max_pk:`